src/shared/models/asset.model.ts

```diff
@@ -0,0 +1,47 @@
+import type BigNumber from 'bignumber.js';
+
+import { WalletBlockchains } from './blockchain.model';
+import { Money } from './money.model';
+
+type WalletAssets = 'native' | 'ft' | 'nft';
```

Comment:
This seems similar to the  `'stx' | 'ft'` type we had before, in that a token can be both. Maybe two divisons would work better.

```ts
type FungibilityType = 'ft' | 'nft';

type TokenType = 'native' | 'non-native'; // not sure on this name
```

Reply:
Sure, I can change.

Reply:
Thinking `FungibleTokenType` would work here?

Reply:
The thinking behind `FungibilityType` was the q "what does one call either a fungible token or non-fungible token?" This kinda sounds like a _type of fungible token_. Not married to fungibility type, but the name shouldn't favour one or the other.

Reply:
`type AssetTypes = 'native-ft' | 'non-native-ft' | 'nft'`

Reply:
sorry I misunderstood, though you were suggesting an alternate to `FungibilityType`. Qualifying `TokenType` with fungible makes perfect sense, if there are no native/non-native nfts.